test_e2e/dest/couchbase.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+set -e
+
+DEST_PATH=$(dirname "$(realpath "$0")")
+SCRIPT_DIR=$(dirname "$DEST_PATH")
+cd "$SCRIPT_DIR"/.. || exit 1
+OUTPUT_FOLDER_NAME=couchbase-dest
+OUTPUT_ROOT=${OUTPUT_ROOT:-$SCRIPT_DIR}
+OUTPUT_DIR=$OUTPUT_ROOT/structured-output/$OUTPUT_FOLDER_NAME
+WORK_DIR=$OUTPUT_ROOT/workdir/$OUTPUT_FOLDER_NAME
+DOWNLOAD_DIR=$OUTPUT_ROOT/download/$OUTPUT_FOLDER_NAME
+max_processes=${MAX_PROCESSES:=$(python3 -c "import os; print(os.cpu_count())")}
+DESTINATION_CB_SCOPE="_default"
+DESTINATION_CB_COLLECTION="_default"
+CI=${CI:-"false"}
+
+source "$SCRIPT_DIR"/env_setup/couchbase/common/constants.env
```

Comment:
Since we're using docker to simulate the connection, can we simply set these env vars in the test file directly? Usually we use this approach when the env vars contain secret values we don't want to expose.